.github/workflows/main.yml

```diff
@@ -53,7 +53,7 @@ jobs:
 
     services:
       postgres:
-        image: postgres:10
+        image: postgres:12.4
```

Comment:
12.4 is 12 and latest. Why 12.4 rather than 12 or latest ?

Reply:
Yeah, 12, 12.4 and latest are tags to the same image. Using `postgres:12` is what makes more sense I think. I'll change it if you agree.

Reply:
I think I'd even agree with "latest" but 12 is sufficiently safe :)

Reply:
I assumed that you'd agree and change it to `postgres:12` :-)